src/assessment/reschedule.py

```diff
@@ -1,23 +1,23 @@
 #!/usr/bin/env python3
-"""PCA Assessment Reschedule to adjust scheduling in the PCA JSON File.
+"""PCA Assessment Reschedule to adjust scheduling in an assessment JSON File.
```

Comment:
We should probably simplify these first lines to just say what the script does- I don't think the `PCA Assessment Reschedule` part adds much value here.  

How does this sound to you?
`Modify campaign start/end dates in an assessment JSON file.`
(Note the present tense of the verb that starts off the sentence.)

If that works for you, please make that change and ensure that the other scripts follow suit.

Reply:
Improve all docopt's first line while moving some additional information into notes on `gophish-test` and `gopish-complete`